addons/account_edi/models/account_move.py

```diff
@@ -617,6 +617,9 @@ def _get_update_invoice_from_attachment_decoders(self, invoice):
     # Business operations
     ####################################################
 
+    def button_process_edi_web_services(self):
+        self.action_process_edi_web_services(with_commit=False)
```

Comment:
```py
            try:
                self.action_process_edi_web_services(with_commit=False)
            except OperationalError as e:
                if e.pgcode == '55P03':
                    raise UserError(_('This document is being sent by another process already. '))
                raise e
```
This error is specific to this button after all

Reply:
I suppose the logging is already part of the cron's management of exceptions and that's why it can be removed ?

Is it OK to slightly change the error message to have better grammar ?

Reply:
Agreed with @smetl 
Since the try/except and detection of that error code is required in the `_process_documents_web_services` method for logging, moving the UserError to the user-specific `button_process_edi_web_services`
- provides no code simplification or shortening
- requires re-throwing the same error just to catch it again

We'll keep this commit simple.